.env.example

```diff
@@ -21,3 +21,28 @@ HOT_WALLET_PRIVATE_KEY=0x2......
 # Settings for force_exit.py
 # EXECUTION_ENDPOINT=https://execution
 # GRAPH_API_URL=https://graph
+
+# Settings for infinity_pools.py
+# EXECUTION_ENDPOINT=https://execution
+
+# MAINNET
+# BTC_WALLET_PRIVATE_KEY=0x...
+# SOL_WALLET_PRIVATE_KEY=0x...
+# SUSDS_WALLET_PRIVATE_KEY=0x...
+# SWISE_WALLET_PRIVATE_KEY=0x...
```

Comment:
Why do we need so many wallets? There should be single wallet used across all the tasks